tests/playwright/favorites-aggregate.spec.ts

```
@@ -0,0 +1,132 @@
+import { expect, test, type Page } from './visual-evidence.fixture';
```

ESLint

[error] Member 'Page' of the import declaration should be sorted alphabetically.

(sort-imports)